Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -26,7 +26,7 @@ try{
 }
 Import-Module -Name CatalystDosIdentity -MinimumVersion $minVersion -Force
 
-$Global:idPSSAppName = "IdentityProviderSearchService"
+$Global:idPSSAppName = "Identity Provider Search Service"
```

Comment:
Switching to spaces to match documenatation/identity service registration

Reply:
I feel like I changed this at some point because it was broken? I remember introducing the global variable, so maybe the bug was the fact that it was mismatched in 2 different scripts, so as long as it's the same everywhere it's fine.

Reply:
Ya there was a bug in the past as there was one script using spaces for the name, and another not using spaces. 
Changing it to spaces since it matches what the docs currently have for the examples.